lldb/test/API/lang/swift/variables/optionals/TestSwiftOptionals.py

```diff
@@ -99,6 +103,27 @@ def do_check_api(self):
             num_children=1)
         uoptString_Some.GetChildAtIndex(99)
 
+        optTrue = self.frame().FindVariable("optTrue")
+        lldbutil.check_variable(
+            self,
+            optTrue,
+            use_dynamic=False,
+            num_children=1)
+
+        optFalse = self.frame().FindVariable("optFalse")
+        lldbutil.check_variable(
+            self,
+            optFalse,
+            use_dynamic=False,
+            num_children=1)
```

Comment:
For future changes, we should look into how to not have children for these cases. In this case there are 3 "scalar" values, there's really no reason for there to be children.